README.md

```diff
@@ -51,47 +51,54 @@ If you're seeing the "Apollo" tab but still having issues, skip ahead to the **D
+
+```bash
+npm run build
+```
+Next, upload the extension to Chrome:
 
 - Open [chrome://extensions](chrome://extensions)
 - Enable the 'Developer Mode' checkbox
 - Click 'Load unpacked extensions...'
-- Select the `apollo-client-devtools/shells/webextension` folder
+- Select the `apollo-client-devtools/` folder
 
-Now, while on any page, open the chrome inspector. If you're inspecting a page that is using Apollo Client, there will be a global `window.__APOLLO_CLIENT__` object on that page. If that object exists, you will see an "Apollo" tab in the inspector menu. This tab will contain the Apollo Client devtools.
+Now, while on any page, open the inspector. If you're inspecting a page that is using Apollo Client, there will be a global `window.__APOLLO_CLIENT__` object on that page. If that object exists, you will see an "Apollo" tab in the inspector menu. This tab will contain the Apollo Client devtools.
 
-### Folder structure
+### Development with Firefox
 
-The extension is built using React and ES6. All the main source code for the devtools exists in the `/src`
-folder, with `devtools/components/Panel.js` being the container component, and `index.js` attatching the
-`Panel` to the document itself. If you're interested in editing the current code or adding a new feature,
-you would do so here.
+To develop extensions with Firefox, we utilize the [web-ext](https://github.com/mozilla/web-ext) command line tool.
 
-Since the devtools are designed to work in multiple environments, the `/shells` folder is where each environment is setup. The one for chrome and firefox live under `webextension`. Wepback bundles the code from the `/src` into `/shells/webextension/dist` when building for chrome for example. To load the
-extension locally, you would load from this folder. Likewise, to upload the extension to the Chrome Webstore,
-you would upload a zip of this folder.
+Running the below command will build and then temporarily install the extension in Firefox:
 
-The root of the repo contains the .bablerc file, webpack config file, and necessary package.json.
+```bash
+npm run firefox
+```
+Now, while on any page, open the inspector. If you're inspecting a page that is using Apollo Client, there will be a global `window.__APOLLO_CLIENT__` object on that page. If that object exists, you will see an "Apollo" tab in the inspector menu. This tab will contain the Apollo Client devtools.
 
-### Reloading the Chrome extension
+### Folder structure
 
-Unfortunately, there is no way to hot-reload a Chrome extension in the inspector while developing it.
+There are two main pieces of the Apollo Devtools: the extension itself and a React application. The extension is the code that communicates with the browser. It's what allows us to search an inspected window for an instance of Apollo Client and to create the Apollo tab in the browser's devtools panel. The React application is what powers the experience what the user clicks on the Apollo tab. 
```

Comment:
The wording seems a bit off in this sentence: "The React application is what powers the experience what the user clicks on the Apollo tab."

Reply:
Nice catch! I'm going to fix the grammatical error there. But otherwise, leave it as-is because this README is 100% going to be revised in the near future.